packages/cli/test/unit/commands/apps/lock.unit.test.ts

```diff
@@ -0,0 +1,37 @@
+import {stdout, stderr} from 'stdout-stderr'
+import * as nock from 'nock'
+import {expect} from 'chai'
+import Cmd  from '../../../../src/commands/apps/lock'
+import runCommand from '../../../helpers/runCommand'
+describe('heroku apps:lock', () => {
+  afterEach(() => nock.cleanAll())
+  it('locks the app', async () => {
+    const apiGetApp = nock('https://api.heroku.com:443')
+      .get('/teams/apps/myapp')
+      .reply(200, {name: 'myapp', locked: false})
+      .patch('/teams/apps/myapp', {locked: true})
+      .reply(200)
+    await runCommand(Cmd, [
+      '--app',
+      'myapp',
+    ])
+
+    expect('').to.eq(stdout.output)
+    expect('Locking myapp...\nLocking myapp... done\n').to.eq(stderr.output)
```

Comment:
This fails on my machine locally. Would you mind updating this to:
```suggestion
    expect(stderr.output).to.contain('Locking myapp... done\n')
```

Reply:
I'm going to hold off on making this change for now, until we get the testing situation straightened out. It is passing on CI.